openpathsampling/netcdfplus/dictify.py

```diff
@@ -359,15 +362,15 @@ def callable_to_dict(c):
                 if module not in ObjectJSON.safe_modules
             ]
 
-            if len(unsafe_modules) > 0:
+            if ObjectJSON.prevent_unsafe_modules and len(unsafe_modules) > 0:
                 if len(err) > 0:
                     err += '\n\n'
 
                 err += 'The function you try to save requires the following' \
                        ' modules to be installed: ' + str(unsafe_modules) + \
                        ' which are not marked as safe! '
                 err += 'You can change the list of safe modules using '
-                err += '\n\n        FunctionCV._safe_modules.extend(['
+                err += '\n\n        ObjectJSON._safe_modules.extend(['
```

Comment:
I think this should be "ObjectJSON.safe_modules.extend([...])" (without the underscore), because that's the variable name in line 44.